commit 2f4024f603832760dfdd3d1b678e0c062adc3bfc
Author: nx111 <gd.zhangdz@gmail.com>
Date:   Fri Feb 28 11:38:54 2014 +0800

    Revert "targa/edison: diabled smartreflex on init again"
    
    This reverts commit 163ba5afd022cf067f76695408c91f97e69a06c6.

diff --git a/arch/arm/mach-omap2/sr_device.c b/arch/arm/mach-omap2/sr_device.c
index 6b0ad70..a092b06 100644
--- a/arch/arm/mach-omap2/sr_device.c
+++ b/arch/arm/mach-omap2/sr_device.c
@@ -156,11 +156,11 @@ static int sr_dev_init(struct omap_hwmod *oh, void *user)
 	}
 
 	sr_set_nvalues(volt_data, sr_data);
-#if defined(CONFIG_MAPPHONE_EDISON) || defined(CONFIG_MAPPHONE_TARGA)
-	sr_data->enable_on_init = false;
-#else
+//#if defined(CONFIG_MAPPHONE_EDISON) || defined(CONFIG_MAPPHONE_TARGA)
+//	sr_data->enable_on_init = false;
+//#else
 	sr_data->enable_on_init = sr_enable_on_init;
-#endif
+//#endif
 
 	od = omap_device_build(name, i, oh, sr_data, sizeof(*sr_data),
 			       omap_sr_latency,
